### PR TITLE
[Keyboard] Adding OSAv2 Topre PCB

### PR DESCRIPTION
**Pull request by blindassassin111**
## Description

Adding the firmware for the OSAv2 topre pcb.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/viktus/osav2_topre/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_split_back_175u_shift(
+        KC_DEL,  KC_ESC,  KC_1, KC_2,    KC_3, KC_4,   KC_5, KC_6, KC_7, KC_8,   KC_9, KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_TILDE,
+        KC_PGUP, KC_TAB,  KC_Q, KC_W,    KC_E, KC_R,   KC_T,       KC_Y, KC_U,   KC_I, KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
+        KC_PGDN, KC_CAPS, KC_A, KC_S,    KC_D, KC_F,   KC_G,       KC_H, KC_J,   KC_K, KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+                 KC_LSFT, KC_Z, KC_X,    KC_C, KC_V,   KC_B,       KC_B, KC_N,   KC_M, KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
```
Comment (zvecr) [inline keyboards/viktus/osav2_topre/keymaps/default/keymap.c:24]: `MO(1)` would go out of bounds.

File: keyboards/viktus/osav2_topre/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_split_back_175u_shift(
+        KC_DEL,  KC_ESC,  KC_1, KC_2,    KC_3, KC_4,   KC_5, KC_6, KC_7, KC_8,   KC_9, KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_TILDE,
+        KC_PGUP, KC_TAB,  KC_Q, KC_W,    KC_E, KC_R,   KC_T,       KC_Y, KC_U,   KC_I, KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSPC,
+        KC_PGDN, KC_CAPS, KC_A, KC_S,    KC_D, KC_F,   KC_G,       KC_H, KC_J,   KC_K, KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+                 KC_LSFT, KC_Z, KC_X,    KC_C, KC_V,   KC_B,       KC_B, KC_N,   KC_M, KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
```
Comment (zvecr) [inline keyboards/viktus/osav2_topre/keymaps/via/keymap.c:24]: Same here.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/viktus/osav2_topre/keymaps/default/keymap.c (hunk 1 above)
Comment (blindassassin111) [reply keyboards/viktus/osav2_topre/keymaps/default/keymap.c:24]: Oops, forgot the second layer.

**Comment by blindassassin111** [comment]
Any changes still needed?

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**